openslides_backend/models/checker.py

```diff
@@ -525,6 +525,28 @@ def check_relation(
                     basemsg,
                     replacement,
                 )
+        elif collection == "motion" and field == "recommendation_extension":
+            RECOMMENDATION_EXTENSION_REFERENCE_IDS_PATTERN = re.compile(
+                r"\[(?P<fqid>\w+/\d+)\]"
+            )
+            recommendation_extension = model["recommendation_extension"]
+            if recommendation_extension is None:
+                recommendation_extension = ""
+
+            possible_rerids = RECOMMENDATION_EXTENSION_REFERENCE_IDS_PATTERN.findall(
+                recommendation_extension
+            )
+            for fqid_str in possible_rerids:
+                re_collection, re_id_ = fqid_str.split(KEYSEPARATOR)
+                if re_collection != "motion":
+                    self.errors.append(
+                        basemsg + f"Found {fqid_str} but only motion is allowed."
+                    )
+                if not self.find_model(re_collection, int(re_id_)):
```

Comment:
Is this check done after the ids are replaced?

Reply:
It is in the checker code and this is done, before the the ids are replaced.

Reply:
Ah yes, nevermind then.